packages/components/table-v2/src/table-v2.tsx

```diff
@@ -273,11 +273,7 @@ const TableV2 = defineComponent({
                     {slots.cell(props)}
                   </Cell>
                 ) : (
-                  <Cell
-                    {...props}
-                    {...tableCellProps}
-                    style={_columnsStyles[props.column.key]}
-                  />
+                  <Cell {...{ ...props, ...tableCellProps }} />
```

Comment:
Maybe we can use `Object.assign` instead, now the code looks a bit weird.
```suggestion
                  <Cell {...Object.assign({}, props, tableCellProps)} />
```

Reply:
I just want to know how this fix will be published in latest version